examples/python/bulk_get_papers_by_pmid/get_papers.py

```diff
@@ -0,0 +1,53 @@
+import csv
+import sys
+
+import requests
+
+
+def chunks(items, chunk_size):
+    return [items[i:i + chunk_size] for i in range(0, len(items), chunk_size)]
+
+
+def fetch_paper_batch(pmids):
+    req = {'ids': [f'PMID:{pmid}' for pmid in pmids]}
+    # https://api.semanticscholar.org/api-docs/graph#tag/Paper-Data/operation/post_graph_get_papers
+    rsp = requests.post('https://api.semanticscholar.org/graph/v1/paper/batch'
+                        '?fields=externalIds,title,authors,year,abstract',
```

Comment:
We can use the `params` parameter instead of formatting the query string by hand.

Reply:
Use a context manager for IO